x-pack/test/functional/fixtures/kbn_archiver/spaces/disabled_features/discover.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I don't think we need to add this new kbn_archive.   It looks like both the kbn_archives that are used in this test only contain a logstash index pattern.  You can just load the same one twice (once in the default space and once in the custom space).